client/app/visualizations/table/columns/boolean.js

```diff
@@ -1,6 +1,54 @@
-/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from 'antd/lib/input';
 import { createBooleanFormatter } from '@/lib/value-format';
 
+function Editor({ column, onChange }) {
+  function handleChange(index, value) {
+    const booleanValues = [...column.booleanValues];
+    booleanValues.splice(index, 1, value);
+    onChange({ booleanValues });
+  }
+
+  return (
+    <React.Fragment>
```

Comment:
Considering those files have the XML notation, shouldn't those files be `.jsx`? :thinking: